backend/routes/shops.js

```diff
@@ -99,25 +98,74 @@ module.exports = function (router) {
     }
   )
 
-  router.get('/shop', async (req, res) => {
-    const { sellerId } = req.session
-
-    if (!sellerId) {
-      return res.status(400).json({ success: false })
+  router.get('/shops', authUser, async (req, res) => {
+    const user = req.seller
+    const order = [['createdAt', 'desc']]
```

Comment:
There is no index on created_at so from a perf standpoint it's better to order by id.